ui/src/app/pages/data/cohort-review/sidebar-content.component.tsx

```diff
@@ -110,10 +114,11 @@ const AnnotationItem = fp.flow(
     };
   }
 
-  componentDidUpdate(prevProps: any): void {
-    const {urlParams: {pid}} = this.props;
+  componentDidUpdate(prevProps: AnnotationProps): void {
+    const location = this.props.location;
+    const {params: {pid}} = matchPath(location.pathname, {path: '/workspaces/:ns/:wsid/data/cohorts/:cid/review/participants/:pid'});
```

Comment:
similar comment to above, I'm wondering if we can swap out a lot of these usages that dig directly into the route to take the value from a prop instead.

In general, I think its OK for props to be passed down even if it can be inferred from the route. it reduces the scope of the simpler components and the dependencies it has.